src/app/public/modules/error/error.component.ts

```diff
@@ -23,6 +23,11 @@ import {
   templateUrl: './error.component.html'
 })
 export class SkyErrorComponent implements OnInit {
+/**
+ * Specifies an error type to use a set of pre-defined values for the image,
+ * title, and description. The valid options are `broken`, `construction`,
+ * `notfound`, and `security`.
```

Comment:
I'd wrap the default values in single quotation marks to indicate `string` values, e.g:
```
`'broken'`, `'construction'`, etc.
```